gulpfile.js

```diff
@@ -18,6 +18,7 @@ gulp.task('default', function () {
 
     // For the second arg see: https://github.com/floatdrop/gulp-watch/issues/242#issuecomment-230209702
     return watch(['source/**/*', '!**/*___jb_tmp___'], function () {
+        console.log(' [PROCESSING] Regenerating');
```

Comment:
Good idea actually! Could you add that to Statie `gulpfile.js`?

Reply:
I see. If you find a good enough way that makes sense, please share it. Otherwise no need to waste time on this.

Reply:
https://github.com/Symplify/Symplify/pull/1365

execSync does not work as expected...